core/src/test/scala/unit/kafka/integration/KafkaServerTestHarness.scala

```diff
@@ -54,11 +58,18 @@ trait KafkaServerTestHarness extends ZooKeeperTestHarness {
 
   protected def securityProtocol: SecurityProtocol = SecurityProtocol.PLAINTEXT
   protected def trustStoreFile: Option[File] = None
+  def clusterAclArgs: Array[String] = Array("--authorizer-properties", 
+                                            s"zookeeper.connect=$zkConnect",
+                                            s"--add",
+                                            s"--cluster",
+                                            s"--allow-principal=$kafkaPrincipal")
```

Comment:
I've had to use this ACL command in the setup of this test harness, so I've needed to use it here, but I'm open to suggestions if there is a way around it.